arkane/statmech.py

```diff
@@ -288,29 +288,29 @@ def load(self, pdep=False):
             except KeyError:
                 raise InputError('Model chemistry {0!r} not found in from dictionary of energy values in species file '
                                  '{1!r}.'.format(self.modelChemistry, path))
-        E0_withZPE, E0 = None, None
+        e0, e_electronic = None, None  # E0 = e_electronic + ZPE
         energyLog = None
         if isinstance(energy, Log) and not isinstance(energy, (GaussianLog, QChemLog, MolproLog)):
             energyLog = determine_qm_software(os.path.join(directory, energy.path))
         elif isinstance(energy, (GaussianLog, QChemLog, MolproLog)):
             energyLog = energy
             energyLog.path = os.path.join(directory, energyLog.path)
         elif isinstance(energy, float):
-            E0 = energy
+            e_electronic = energy
         elif isinstance(energy, tuple) and len(energy) == 2:
             # this is likely meant to be a quantity object with ZPE already accounted for
-            energy_temp = Quantity(energy)
-            E0_withZPE = energy_temp.value_si  # in J/mol
+            energy = Quantity(energy)
+            e_0 = energy.value_si  # in J/mol
         elif isinstance(energy, tuple) and len(energy) == 3:
-            if energy[2] == 'E0':
-                energy_temp = Quantity(energy[:2])
-                E0 = energy_temp.value_si / constants.E_h / constants.Na  # convert J/mol to Hartree
-            elif energy[2] == 'E0-ZPE':
-                energy_temp = Quantity(energy[:2])
-                E0_withZPE = energy_temp.value_si  # in J/mol
+            if energy[2].lower() in ['e_elect', 'e_electronic', 'electronic_energy']:
```

Comment:
Can we try to make the naming consistent so that there aren't three different variable names that all correspond to the same thing?

Reply:
Sure